### ISSUE TITLE
ING costs have changed

### ISSUE THREAD

**Issue by Virtlink** [nicwortel/indexfondsenvergelijken.nl#10]
I don't know when this happened, but it seems the costs for investing at ING have changed. Their page at https://www.ing.nl/particulier/beleggen/beginnen-met-beleggen/tarieven/tarieven-zelf-op-de-beurs.html now lists:

```
Basisfee:         € 20,- per jaar
t/m € 100.000     0,24%
t/m € 500.000     0,12%
daarboven         0,06%
```

The website, however, still shows:

```
Basisfee:         € 5,- per kwartaal
t/m € 100.000     0,06%
t/m € 500.000     0,03%
daarboven         0,015%
```

**Comment by gerbenvl** [comment]
But that's the same right? Per kwartaal / per jaar?

**Comment by Virtlink** [comment]
But the percentages changed too. They are four times higher now, which can really influence the calculation.

**Comment by gerbenvl** [comment]
In 2020 I had the same percentages in my data. Nothing changed there as far as I can see.

**Comment by Virtlink** [comment]
Okay then I misinterpreted the data shown on the website.

**Comment by nicwortel** [comment]
Hi @Virtlink, thank you for raising this issue. I see you have already closed it but I would still like to provide some explanation.

I have recently rewritten the service fees logic, see commit https://github.com/nicwortel/indexfondsenvergelijken.nl/commit/9ac22dfe5488e6376243d76f27998383131d7342. I used to store (and display) the yearly fees, but now I store the quarterly / monthly fees (so the actual fees that are being applied in the simulation).
As @gerbenvl noted, the quarterly fees are all simply 25% of the yearly fees. So ING's costs haven't changed.

Some banks show quarterly or monthly fees on their website, and other banks (such as ING) show yearly fees even though they actually charge costs every quarter. I've decided to display the actual fee on my website, together with the frequency in which it is charged. However, I can imagine that this may be confusing, so I'm open to suggestions as to how this may be displayed in a more intuitive way.

There is actually even more complexity, as banks have different ways to calculate your average invested capital over a certain period. This is taken into account in the simulation, but not yet displayed in the costs tab.
